lib/resources/r4/financial_transaction.yaml

```diff
@@ -17,8 +16,7 @@ fields:
 
 - name: identifier
   required: 'Yes'
-  cardinality: 0..*
-  type: Identifier
+  type: List of Identifier
```

Comment:
double check here, and most extensions

Reply:
After reading the notes provided on the fields and checking verifies, I think these are correct.